tox.ini

```diff
@@ -2,25 +2,54 @@
 minversion = 1.9
 envlist =
     lint
-    py{37,38,39,py3}-django22-alchemy-mongoengine
```

Comment:
`alchemy` and `mongoengine` were always used without fail, removed it from the env name, and its now always used inside `deps`

Reply:
As a secondary thought, maybe have it as a separate thing entirely is beneficial, we really do want to test that if django is not installed `factory boy` continues to work as expected. Same goes for `sqlalchemy` -- but this is a bit out of scope at the moment. This change adds nothing / removes nothing from what was there originally. Food for thought.

Reply:
IMO, leaving as-is is fine. I agree that testing the import without optional components would be beneficial, but certainly not necessary for the work you are undertaking.

Reply:
Lets continue this conversation over here: https://github.com/FactoryBoy/factory_boy/pull/940